crates/primitives/src/account.rs

```diff
@@ -4,8 +4,12 @@ use bytes::Buf;
 use reth_codecs::Compact;
 use revm_primitives::JumpTable;
 use serde::{Deserialize, Serialize};
+#[cfg(feature = "std")]
 use std::ops::Deref;
 
+#[cfg(not(feature = "std"))]
+use core::ops::Deref;
```

Comment:
Please just use `core` as it's always available.

Reply:
for any case where we have both std/core import for the same thing?

Reply:
yes, std just re-exports core internally for most items